### ISSUE TITLE
Train RNN-T ASR model on 960h Librispeech? 

### ISSUE THREAD

**Issue by scufan1990** [espnet/espnet#3867]
Hi,
There is a problem about training  a  conformer+RNN-T model.
How about the cer and wer with one GPU?
i'm train the model on one RTX TITAN GPU, training the conformer(encoder layers 16, encoder dim 144, decoder layer 1, decoder dim 320), after 50 epoch training the CER is about 27 and don't reduce anymore.

**Comment by scufan1990** [comment]
i train the model on Librispeech 960h

**Comment by sw005320** [comment]
Can you paste the learning rate?
BTW, I can actually train the conformer+RNN-T model by only tuning the learning rate.

**Comment by b-flo** [comment]
Hi,

> How about the cer and wer with one GPU?

If I'm not mistaken, we only have results for RNN-T (trained with 1x RTX8000 45Gb). The best one being:
 
| dev-clean | dev-other | test-clean | test-other |
| - | - | - | - |
| 3.8 | 10.8 | 4.0 | 10.9 |

For Conformer-T, we trained with 2 or 4 GPUs (depending on the number of BPE units). Current results are, without LM:

| dev-clean | dev-other | test-clean | test-other |
| - | - | - | - |
| 2.7 | 6.7 | 2.9 | 6.8 |

@sw005320 I'm confused, didn't we add Transducer configs and results for Librispeech to ESPnet?
The pre-trained model needs to be updated, I'll ask for some GPUs for next week and do the training if possible.

**Comment by sw005320** [comment]
I did not add it simply because I forgot it.
Sorry.
I'll do it in the next week...

**Comment by scufan1990** [comment]
> 
I found the results for Librispeech to ESPnet , but i don't know  the result train on one GPU(RTX TITAN 24G)
And i'm confused the RNN-T model (training on one GPU)isn't get  close performance with the result.


> Hi,
> 
> > How about the cer and wer with one GPU?
> 
> If I'm not mistaken, we only have results for RNN-T (trained with 1x RTX8000 45Gb). The best one being:
> 
> dev-clean	dev-other	test-clean	test-other
> 3.8	10.8	4.0	10.9
> For Conformer-T, we trained with 2 or 4 GPUs (depending on the number of BPE units). Current results are, without LM:
> 
> dev-clean	dev-other	test-clean	test-other
> 2.7	6.7	2.9	6.8
> @sw005320 I'm confused, didn't we add Transducer configs and results for Librispeech to ESPnet? The pre-trained model needs to be updated, I'll ask for some GPUs for next week and do the training if possible.

Thank you!

I found the results for Librispeech to ESPnet , but i don't know  the result train on how many GPU . And i'm confused the RNN-T model (training on one GPU)isn't get  close performance with the result.   

Thank you for your detail reply, i get it. I will try it later .

**Comment by marcoyang1998** [comment]
> Hi,
> 
> > How about the cer and wer with one GPU?
> 
> If I'm not mistaken, we only have results for RNN-T (trained with 1x RTX8000 45Gb). The best one being:
> 
> dev-clean	dev-other	test-clean	test-other
> 3.8	10.8	4.0	10.9
> For Conformer-T, we trained with 2 or 4 GPUs (depending on the number of BPE units). Current results are, without LM:
> 
> dev-clean	dev-other	test-clean	test-other
> 2.7	6.7	2.9	6.8
> @sw005320 I'm confused, didn't we add Transducer configs and results for Librispeech to ESPnet? The pre-trained model needs to be updated, I'll ask for some GPUs for next week and do the training if possible.

Hi, could you please tell me which config was used for training the Conformer-T model?

**Comment by eesungkim** [comment]
Hi, @marcoyang1998 @sw005320

The performance of the conformer-transducer model in ESPNET1 was not good recently, but after the [bug](https://github.com/espnet/espnet/pull/4020) was fixed, the following performance was confirmed with recent version of ESPNET. 

https://github.com/eesungkim/espnet/blob/master/egs/librispeech/asr1/RESULTS.md

I hope it helps for you.

**Comment by marcoyang1998** [comment]
> Hi, @marcoyang1998 @sw005320
> 
> The performance of the conformer-transducer model in ESPNET1 was not good recently, but after the [bug](https://github.com/espnet/espnet/pull/4020) was fixed, the following performance was confirmed with recent version of ESPNET.
> 
> https://github.com/eesungkim/espnet/blob/master/egs/librispeech/asr1/RESULTS.md
> 
> I hope it helps for you.

So based on the numbers in [this](https://github.com/eesungkim/espnet/blob/master/egs/librispeech/asr1/RESULTS.md) result file, it still has a gap with the original conformer paper right? The original paper reports a WER of 2.1 and 4.3 on test-clean and test-other with Conformer(L)

**Comment by b-flo** [comment]
> So based on the numbers in [this](https://github.com/eesungkim/espnet/blob/master/egs/librispeech/asr1/RESULTS.md) result file, it still has a gap with the original conformer paper right? The original paper reports a WER of 2.1 and 4.3 on test-clean and test-other with Conformer(L)

The Transducer model @eesungkim trained is not comparable to Gulati et al's Conformer(L).
After we finish some things related to the ESPnet2 implementation, we will work on improving our Librispeech model.
